scripts/anvil_tools/README.md

```diff
@@ -0,0 +1,101 @@
+# AnVIL Tools/Scripts
+
+------------------------
+#### This directory contains scripts/tools to automate steps in the AnVIL data delivery workspace creation process.
+------------------------
+
+### Setup
+    Follow the steps below to set up permissions prior to running any of the listed scripts:
+    1. `gcloud auth login username@broadinstitute.org`
+    2. `gcloud auth application-default login`
```

Comment:
not necessary, but if desired, you can streamline this by combining to one line: `gcloud auth login username@broadinstitute.org; gcloud auth application-default login`